### ISSUE TITLE
Take a look at background colors in ffe-grid

### ISSUE THREAD

**Issue by ivarni** [SpareBank1/designsystem#256]
#### Which package is this issue related to?

`ffe-grid`, `ffe-grid-react`

#### Describe your issue (screenshots welcome!)

[The designguide](http://design.sparebank1.no/visuell-identitet.html#visuell-identitet_farger) defines a series of colours that can be used as backgrounds, namely

> ffe-white
ffe-grey-warm
ffe-sand
ffe-green-mint
ffe-blue-pale
ffe-orange-salmon
ffe-red

and

> ffe-blue-sky
ffe-blue-pale
ffe-blue-ice

[The grid](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-grid-react/src/background-colors.js) supports the following

> ffe-blue-cobalt
ffe-blue-ice
ffe-blue-pale
ffe-blue-royal
ffe-green-mint
ffe-grey-cloud
ffe-purple-magenta
ffe-sand

Some of those we support do not belong, some of those that belong we do not support

To sum up:

These should possibly be removed:
> ffe-blue-cobalt
ffe-blue-royal
ffe-purple-magenta

These should be added:
> ffe-grey-warm
ffe-orange-salmon
ffe-red
ffe-blue-sky

I propose that we remove those that are not listed as background colours in order to discourage them being used and leave it to user-land to add those in projects where there are some kind of special reason to allow them and that we add those that are missing.

**Comment by ivarni** [comment]
I am going to go ahead and add the missing ones at least and I'll wait with removing anything untill we can figure out if we want to do that.

**Comment by selbekk** [comment]
cc @adidick - do you agree that we remove the background colors specified above? What should they be migrated to?

**Comment by ivarni** [comment]
I don't think we need to decide right here what they should be migrated to, consumers using them can choose to either use something else or add their own CSS for this, surely?

**Comment by selbekk** [comment]
I know they can solve this in different ways, but I would greatly appreciate it as a consumer if somebody told me "hey, you're using the wrong light blue background. Use this one instead" while updating.

**Comment by adidick-zz** [comment]
Sorry, I've been away since Wednesday. 
I'm all for adding colours that are missing, but as for removing, I'm less sure.
For example, blue-royal is used as the background colour for the footer (and possibly other things?), so I wonder if it's necessary to remove the others, or maybe have some way of prioritising/recommending the most appropriate ones and having a secondary set of edge-case colours? Or like you guys have said, remove them and make it up to projects to make the decisions themselves. 
My point is, can we discourage things from being used but still make them available for those that need them in these edge-cases? It should be ultimately up to the designers to make the right choice, but we all know how that can go sometimes...

**Comment by ivarni** [comment]
> My point is, can we discourage things from being used but still make them available for those that need them in these edge-cases?

Yes we can. That's exactly what we do if we drop the FFE support for it. They can still use their own CSS if they really really really want to do it. If want to we discourage doing something then not making it too easy to do is a really good way to signal that in my opinion.

**Comment by ivarni** [comment]
@selbekk I *was* planning on mentioning it in the changelog :)
